### PR TITLE
WebAPI: Update Method pages to modern structure (part 11)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #14857 

## Summary
Updating the method pages to follow the current templates:
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods
https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template

Changes Include:
- fix syntax section
- changing section titles to suggested titles in the template
- change `{{DOMxRef("DOMString")}}` to `string`
- remove extra spaces in between words and extra trailing spaces
- other small corrections

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/htmltablerowelement/insertcell/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ reference to the cell.
 ## Syntax
 
 ```js
-var newCell = HTMLTableRowElement.insertCell(index);
+insertCell(index)
```
Comment (teoli2003) [inline files/en-us/web/api/htmltablerowelement/insertcell/index.md:30]:
```suggestion
insertCell()
insertCell(index)
```

File: files/en-us/web/api/htmltableelement/insertrow/index.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ let row = specific_tbody.insertRow(index)
 ## Syntax
 
 ```js
-var newRow = HTMLTableElement.insertRow(index);
+insertRow(index)
```
Comment (teoli2003) [inline files/en-us/web/api/htmltableelement/insertrow/index.md:36]:
```suggestion
insertRow()
insertRow(index)
```

File: files/en-us/web/api/htmlvideoelement/msinsertvideoeffect/index.md [hunk 3]
```diff
@@ -19,13 +19,13 @@ This proprietary method is specific to Internet Explorer and Microsoft Edge.
 ## Syntax
 
 ```js
-str = HTMLMediaElement.msInsertVideoEffect(activatableClassId: {{DOMxRef("DOMString")}}, effectRequired: {{JSxRef("Boolean", "boolean")}}, config);
+msInsertVideoEffect(activatableClassId, effectRequired, config)
```
Comment (teoli2003) [inline files/en-us/web/api/htmlvideoelement/msinsertvideoeffect/index.md:22]:
```suggestion
msInsertVideoEffect(activatableClassId, effectRequired)
msInsertVideoEffect(activatableClassId, effectRequired, config)
```

File: files/en-us/web/api/idbcursor/continue/index.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ advances to the immediate next position, based on its direction.
 ## Syntax
 
 ```js
-cursor.continue(key);
+continue(key)
```
Comment (teoli2003) [inline files/en-us/web/api/idbcursor/continue/index.md:27]:
```suggestion
continue()
continue(key)
```

File: files/en-us/web/api/idbdatabase/createobjectstore/index.md [hunk 5]
```diff
@@ -29,8 +29,8 @@ transaction.
 ## Syntax
 
 ```js
-IDBDatabase.createObjectStore(name);
-IDBDatabase.createObjectStore(name, options);
+createObjectStore(name)
+createObjectStore(name, options)
 ```
 
 ### Parameters
```
Comment (teoli2003) [inline files/en-us/web/api/idbdatabase/createobjectstore/index.md:36]: _optionalParameters_ should be renamed as _options_ in the list below

File: files/en-us/web/api/idbindex/get/index.md [hunk 6]
```diff
@@ -28,7 +28,7 @@ with.
 ## Syntax
 
 ```js
-var request = myIndex.get(key);
+get(key)
```
Comment (teoli2003) [inline files/en-us/web/api/idbindex/get/index.md:31]:
```suggestion
get()
get(key)
```

File: files/en-us/web/api/idbindex/getkey/index.md [hunk 7]
```diff
@@ -27,7 +27,7 @@ Note that this doesn't return the whole record as {{domxref("IDBIndex.get")}} do
 ## Syntax
 
 ```js
-var request = myIndex.getKey(key);
+getKey(key)
```
Comment (teoli2003) [inline files/en-us/web/api/idbindex/getkey/index.md:30]:
```suggestion
getKey()
getKey(key)
```

File: files/en-us/web/api/idbobjectstore/delete/index.md [hunk 8]
```diff
@@ -27,9 +27,8 @@ record — without having to explicitly look up the record's key.
 ## Syntax
 
 ```js
-var request = objectStore.delete(Key);
-
-var request = objectStore.delete(KeyRange);
+delete(Key)
+delete(KeyRange)
```
Comment (teoli2003) [inline files/en-us/web/api/idbobjectstore/delete/index.md:31]: Either we separate _key_ and _keyrange_ in the parameter list, or we keep them together and we have here:
```suggestion
delete(Key)
```

File: files/en-us/web/api/imagedecoder/decode/index.md [hunk 9]
```diff
@@ -16,7 +16,7 @@ The **`decode()`** method of the {{domxref("ImageDecoder")}} interface enqueues
 ## Syntax
 
 ```js
-ImageDecoder.decode(options)
+decode(options)
```
Comment (teoli2003) [inline files/en-us/web/api/imagedecoder/decode/index.md:19]:
```suggestion
decode()
decode(options)
```

File: files/en-us/web/api/lockedfile/getmetadata/index.md [hunk 10]
```diff
@@ -19,7 +19,7 @@ file.
 ## Syntax
 
 ```js
-var request = instanceOfLockedFile.getMetadata(param);
+getMetaData(param)
```
Comment (teoli2003) [inline files/en-us/web/api/lockedfile/getmetadata/index.md:22]:
```suggestion
getMetaData()
getMetaData(param)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
Looks like these are very old pages. They haven't marked optional params with square brackets. 😕

File: files/en-us/web/api/idbobjectstore/delete/index.md [hunk 11]
```diff
@@ -28,7 +28,7 @@ record — without having to explicitly look up the record's key.
 
 ```js
 delete(Key)
```
Comment (teoli2003) [inline files/en-us/web/api/idbobjectstore/delete/index.md:30]: Not twice the same entry
```suggestion
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Still a tiny last change!

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**